wolfcrypt/src/port/kcapi/kcapi_ecc.c

```diff
@@ -190,14 +192,52 @@ int KcapiEcc_Sign(ecc_key* key, const byte* hash, word32 hashLen, byte* sig,
         }
     }
     if (ret == 0) {
-        ret = kcapi_akcipher_sign(key->handle, hash, hashLen, sig, *sigLen,
+        if ((long int)sig % sysconf(_SC_PAGESIZE) != 0) {
```

Comment:
Isn't `size_t` is the right type to use here? Why not just call `sysconf(_SC_PAGESIZE)` once and set to local variable?

Reply:
sysconf() returns a long, not a size_t. Returning it to a local makes sense.

Reply:
Since `sig` is a point the `long int` on 32-bit will have a type size mismatch. Using `size_t` is portable. You may also need to cast the result of page size to `size_t`.